app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -800,9 +800,10 @@ export function PlaygroundDatasetExamplesTable({
                   aria-label="View example details"
                   icon={<Icon svg={<Icons.ExpandOutline />} />}
                   onPress={() => {
-                    navigate(
-                      `/playground/datasets/${datasetId}/examples/${row.original.id}`
-                    );
+                    setSearchParams((prev) => {
+                      prev.set("exampleId", row.original.id);
+                      return prev;
+                    });
```

Comment:
are you trying to have it replace? not familiar with this syntax

Reply:
oh nvm

Reply:
it just works like useState. probably not necessary in this case, but better safe than sorry